data/src/main/java/org/akvo/flow/data/entity/form/FormMapper.kt

```diff
@@ -25,6 +25,42 @@ import javax.inject.Inject
 
 class FormMapper @Inject constructor() {
 
+    fun mapForms(cursor: Cursor?): List<DataForm> {
+        val forms = mutableListOf<DataForm>()
+        if (cursor != null && cursor.moveToFirst()) {
+            do {
+                val id = getIntColumnValue(cursor, SurveyColumns._ID)
+                val formId = getStringColumnValue(cursor, SurveyColumns.SURVEY_ID)
+                val surveyId = getIntColumnValue(cursor, SurveyColumns.SURVEY_GROUP_ID)
```

Comment:
Also wondering why the difference  i.e this is a `getInt` and the formId is a `getString`

Reply:
because one of them is saved as int value and the other as string in the database